spec/models/assessment_spec.rb

```diff
@@ -258,4 +259,23 @@
       it { is_expected.to include("request_further_information") }
     end
   end
+
+  describe "#selected_failure_reasons_empty?" do
+    subject(:selected_failure_reasons_empty?) do
+      assessment.selected_failure_reasons_empty?(assessment)
+    end
+
+    context "with a decline with no failure reasons" do
+      before { create(:assessment, :decline) }
```

Comment:
This line isn’t changing the test as it’s not overwriting the `assessment` variable defined near the top of the file, and therefore not changing the `assessment` in the subject.